spec/features/user_posts_index_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'User Post Index Page', type: :feature do
+  let(:user) { User.first }
+  before do
+    11.times { |i| Post.create(author: user, title: "Post #{i}", text: "Content of post #{i}") }
+    visit user_posts_path(user)
+  end
```

Comment:
- [ ] Please kindly fix the failing tests screenshot below 👇 , perhaps you need to declare value for the user because I think `User.first` will definetely reference an empty instance of variable. The error indicates that `@user` is nil, and Rails is unable to generate the route because it requires a non-nil value for user_id. 

<img width="611" alt="deel" src="https://github.com/GeekyHacks/FlightOperationsBlog/assets/103386183/d8981394-cba2-4e42-8941-e31b1a53db69">